lib/helper/FileSystem.js

```diff
@@ -80,37 +100,51 @@ class FileSystem extends Helper {
   /**
    * Checks that file found by `seeFile` includes a text.
    * @param {string} text
-   * @param {string} encoding
+   * @param {?string} [encoding='utf8'] (optional, `utf8` by default)
    */
-  seeInThisFile(text, encoding) {
+  seeInThisFile(text, encoding = 'utf8') {
     const content = getFileContents(this.file, encoding);
     fileIncludes(this.file).assert(text, content);
   }
 
   /**
    * Checks that file found by `seeFile` doesn't include text.
    * @param {string} text
-   * @param {string} encoding
+   * @param {?string} [encoding='utf8'] (optional, `utf8` by default)
    */
-  dontSeeInThisFile(text, encoding) {
+  dontSeeInThisFile(text, encoding = 'utf8') {
     const content = getFileContents(this.file, encoding);
     fileIncludes(this.file).negate(text, content);
   }
 
   /**
    * Checks that contents of file found by `seeFile` equal to text.
    * @param {string} text
-   * @param {string} encoding
+   * @param {?string} [encoding='utf8'] (optional, `utf8` by default)
    */
-  seeFileContentsEqual(text, encoding) {
+  seeFileContentsEqual(text, encoding = 'utf8') {
     const content = getFileContents(this.file, encoding);
     fileEquals(this.file).assert(text, content);
   }
 
+  /**
+   * Checks that contents of the file found by `seeFile` equal to contents of the file at `pathToReferenceFile`.
+   * @param {string} pathToReferenceFile
+   * @param {?string} [encoding='utf8'] (optional, `utf8` by default)
+   * @param {?string} [encodingReference='utf8'] (optional, `utf8` by default)
+   */
+  seeFileContentsEqualReferenceFile(pathToReferenceFile, encoding = 'utf8', encodingReference = '') {
+    const content = getFileContents(this.file, encoding);
+    assert.ok(fileExists(pathToReferenceFile), `Reference file ${pathToReferenceFile} not found.`);
+    encodingReference = encodingReference || encoding;
+    const expectedContent = getFileContents(pathToReferenceFile, encodingReference);
+    fileEquals(this.file).assert(expectedContent, content);
+  }
+
   /**
    * Checks that contents of file found by `seeFile` doesn't equal to text.
    * @param {string} text
-   * @param {string} encoding
+   * @param {?string} [encoding='utf8'] (optional, `utf8` by default)
```

Comment:
`?` is unnecessary there and in other added functions

`[param=defaultValue]` is enough to say that arguments is optional.
`null` is not waited in this and others function